scripts/init-celestia.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+GENESIS_FILE="/home/celestia/.celestia-appd/config/genesis.json"
```

Comment:
[Not blocking] saw a couple PRs to celestia-app to standardize on `./celestia-app` so just flagging that this still uses `./celestia-appd`

```suggestion
GENESIS_FILE="/home/celestia/.celestia-app/config/genesis.json"
```

Reply:
100%, I had made a note of this also. We can rebase the feat branch that Manav made which I took the docker build from and pick up the changes here. 

Let's do that before merging this PR. I can look into it

Reply:
I'll just merge this and create a follow up